public/js/components/SortingBar.jsx

```diff
@@ -9,18 +9,36 @@ export default class SortingBar extends React.Component {
     ArticleActions.fetchArticles(sort)
   }
 
+  constructor() {
+    super()
+    this.state = {
+      newActive: null,
```

Comment:
The initial sort is `NEW`, so this initial value should be `true`. Also if the values should be `boolean`s I don't think you should be setting them to `null`